plugins/hls-refactor-plugin/hls-refactor-plugin.cabal

```diff
@@ -86,6 +86,9 @@ library
     , lens
     , data-default
     , time
+    -- FIXME: Only needed to workaround for qualified imports in GHC 9.4
```

Comment:
Do you want to guard the inclusion of these with a cabal conditional to make it explicit?

Reply:
I think that's not worth the trouble. What are a couple of not 100% necessary dependencies that we likely depend on already, in comparison to a cabal flag?

Reply:
Given that `regex-applicative` is stable and somewhat lightweight dependency, I would leave the dependency as-is.